backend/src/game/cards/Card_JSON_Parser.py

```diff
@@ -0,0 +1,56 @@
+import json
+from enum import Enum
+from typing import List
+
+with open('backend\\src\\game\\cards\\KOTCards.json', "r") as json_file:
+    card_file = json.load(json_file)
+
+
+class CardType(Enum):
+    keep = 1
+    discard = 2
+    invalid = 99
+
+
+class Card(object):
+    def __init__(self, name, cost, card_type, effect, footnote):
+        self.name = name
+        self.cost = cost
+        self.effect = effect
+        self.footnote = footnote
+        if str.lower(card_type) == "keep":
+            self.card_type = CardType.keep
+        elif str.lower(card_type) == "discard":
+            self.card_type = CardType.discard
+        else:
+            self.card_type = CardType.invalid
```

Comment:
You must assert that card_type is type str otherwise you could potentially generate an error if someone passed in an int.  Also would suggest making this block it's own function

Reply:
Ok I added more assertions to the is_valid method. Would that satisfy this requirement or should the assertions be in the init? Also extracted the block I think you referenced to it's own funciton